deploy/cr.yaml

```diff
@@ -171,6 +171,10 @@ spec:
         cpu: "300m"
         memory: "0.5G"
     volumeSpec:
+#      annotations:
+#        service.beta.kubernetes.io/aws-load-balancer-backend-protocol: http
```

Comment:
please try to use examples more specific for PVC like 'volume.beta.kubernetes.io/storage-class=example-hostpath'

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/1096/commits/9401698f2f4b4176a21dccff943eb01c2e7b632a